src/windows/models/credits_model.py

```diff
@@ -56,7 +56,7 @@ def update_model(self, filter=None, clear=True):
 
         for person in self.credits_list:
             # Get details of person
-            data = defaultdict("")
+            data = defaultdict(list)
```

Comment:
Hmm. This is a tricky one. I never noticed that `icons` was a _list_ of characters, in the JSON file, which would seem to support a `list` default.

<details>
<summary>Digression</summary>

If you look at the comprehension on 84:
```python
for contrib in [n for n in self.icon_mapping if n in data["icons"]]
```
I suspect `data["icons"]` was originally intended to be a string, and the icons would be single-character flags packed into that string. IOW, the idea was that `data["icons"]` would contain e.g. `"knd"` for a kickstarter contributor and Patreon supporter who's also a developer. The comprehension would match each of those characters, and it would display all of the appropriate icons.

Of course, whoever wrote this never got that to work right, because you can't set multiple icons for one cell of a data model. So then they added that `s` override for "multiple contributions". And then `"icons"` got turned into a list, because it's actually easier to export a list of characters than a packed string.
</details>

But the code also contains a lot of calls like `data["name"].lower()`, which would break on a list item. So, I suspect this should actually be `defaultdict(str)`. Which will work for `data["icons"]` as well, since an empty string is still compatible with the `in` usage in the comprehension.

```suggestion
            data = defaultdict(str)
```

Just for fun, I notice now that there are also several rows in the  JSON with a `"t"` in the icons list. What does `"t"` mean? Search me, it's not defined anywhere in the code. I suspect they might be unused translator credits. Those ended up getting pulled from the translation file itself, instead.

Reply:
Heh. Oh, the `defaultdict` was me. (As was the `icon_mapping`, so the bug here is definitely my fault.)

Well, then I can say confidently that it _should_ be `defaultdict(str)`, because the previous code (which was convoluted and verbose, but basically equivalent) definitely treated the `data` values as strings. I just misunderstood how to create a `defaultdict`, apparently.

Reply:
Ok I replaced the "list" by "str" in the PR. Everythings seems to work fine. I didn't notice the "t" before you mention it so I have no idea what it stands for...